### PR TITLE
[hot-fix] increase data-node memory limits to 4Gi

### PR DESCRIPTION
**Pull request by hao1939**
Increase data-node memory limits to 4Gi.

### CODE REVIEW

File: src/hadoop-data-node/deploy/hadoop-data-node.yaml.template [hunk 1]
```diff
@@ -52,6 +52,8 @@ spec:
           periodSeconds: 3
         resources:
           limits:
+            memory: "4Gi"
+          requests:
             memory: "1Gi"
         env:
         - name: HADOOP_DATANODE_OPTS
```
Comment (mzmssg) [inline src/hadoop-data-node/deploy/hadoop-data-node.yaml.template]: Need to raise JVM memory opts?

**Comment by mzmssg** [comment]
I think we also need to modify here?
https://github.com/Microsoft/pai/blob/6cfcb8a3553a6142ebde8a6a07f4cda6d7239179/src/hadoop-node-manager/deploy/hadoop-node-manager-configuration/nodemanager-generate-script.sh#L66

File: src/hadoop-data-node/deploy/hadoop-data-node.yaml.template (hunk 1 above)
Comment (hao1939) [reply src/hadoop-data-node/deploy/hadoop-data-node.yaml.template]: Not sure, currently we haven't found any java limit was hit.

**Comment by hao1939** [comment]
@mzmssg ,

It's node manager limit. How much should it be?

File: src/hadoop-data-node/deploy/hadoop-data-node.yaml.template (hunk 1 above)
Comment (mzmssg) [reply src/hadoop-data-node/deploy/hadoop-data-node.yaml.template]: #1658 For 175node screenshot, jave OOM is recorded. So I think a larger JVM opts is needed.

Comment (hao1939) [reply src/hadoop-data-node/deploy/hadoop-data-node.yaml.template]: Hi @mzmssg , 
What about increase the max heap size to 2G?

**Comment by hao1939** [comment]
Hi @mzmssg , increase the reserved memory on worker to 16G.

**Pull request merged**